WaterLog_Backend/Controllers/SegmentLeaksController.cs

```diff
@@ -61,24 +61,28 @@ public async Task<ActionResult<string>> GetLitres(int id)
 
         //Resolve Leakage
         [HttpPut("resolve/{id}")]
-        public async Task<IActionResult> Resolve(int id)
+        public async Task<ActionResult<IEnumerable<SegmentLeaksEntry>>> Resolve(int id)
```

Comment:
resolve what? more descriptive function names

Reply:
It's an API for resolving leaks, just changed to resolveLeak

Reply:
you didnt push.

Reply:
Don't know why the changes were not made, pushed again